### ISSUE TITLE
Account Signup page: Problem with Link to "Terms and Conditions"

### ISSUE THREAD

**Issue by BCcLeva** [BCcampus/tlpd#128]
### Description 
On the Cert account signup page (https://tlpddev.bccampus.ca/sign-up/) the link to "Terms and Conditions" opens a popup that has no content.

![screen shot 2018-12-04 at 3 35 09 pm](https://user-images.githubusercontent.com/43043264/49480111-507b7700-f7da-11e8-95e1-f8a6d7cb186c.png)

![screen shot 2018-09-28 at 12 11 14 pm](https://user-images.githubusercontent.com/34721375/46228956-b1895900-c318-11e8-9539-31e25adc98bd.png)

The link label should be "Terms of Use" instead of "Terms and Conditions"

[Description of the bug or feature]

The text for the terms of use should be in the modal pop-up when link is clicked.

Link text to pop-up the modal should be "Terms of Use" rather than "Terms and Conditions"

**Actual behaviour** 

[What actually happened]

### Steps to reproduce the problem

https://tlpddev.bccampus.ca/sign-up/

Click Terms and Conditions

### System Information

- TLPD: [Enter Version here] 
- WordPress: [Enter WP Version number here]

**Comment by paulagaube** [comment]
This is on the Sign-Up page, here: https://tlpddev.bccampus.ca/sign-up/

Under "Agreement Terms: (Required)" 
Under the Checkbox: I Agree
The Text "Terms and Conditions" is going to this URL: https://tlpddev.bccampus.ca/sign-up/#terms

I went to modify this URL from the Dashboard but cannot see how to update the URL. Was on this Buddy Press page:
Dashboard > Users > Profile Fields > Agreement Terms

Maybe the content is hard-coded so the text appears in the modal? Or I'm looking in the wrong place, I'm not sure. I hope @bdolor knows, @BCcLeva  

On TLPDDEV, there is a Page with "Terms of Use", here: https://tlpddev.bccampus.ca/terms-and-conditions/

So I created a new page on TLPD (Prod) with the same content, but called it "Terms of Use", here: https://tlpd.bccampus.ca/terms-of-use/

**Comment by BCcLeva** [comment]
I had a discussion with Brian Case and he in turn with Denise a few days ago and we are to use the Terms and Conditions from the BCcampus.ca site. I have updated the https://tlpd.bccampus.ca/terms-of-use/ page on TLPD PROD and will do so on TLPD DEV

**Comment by paulagaube** [comment]
I started working on this, but was side-tracked by something outside the sprint. Would like to revisit this later. Putting back into backlog for now.

**Comment by paulagaube** [comment]
@bdolor We need your help to finish this issue please. See the image below. The text showing on the Sign Up page for the "Agreement Terms" says "Terms and Conditions" instead of "Terms of Use" and is hard-coded on the page so I can't edit it. The content of the Page with the terms is not showing up in the Modal window either.
Also see original issue above for details. The page that has the correct text is linked in the footer URL for "Terms of Use", here: https://tlpddev.bccampus.ca/terms-of-use/ (CERT) or https://tlpd.bccampus.ca/terms-of-use/ (PROD).

<img width="598" alt="signup-page-agreementterms" src="https://user-images.githubusercontent.com/26391359/49977637-ef4f5400-fefb-11e8-85c5-f1688c32979c.png">

**Comment by bdolor** [comment]
ca c'est fini

**Comment by paulagaube** [comment]
Thank you! This looks lovely now. Tested on PC on Chrome, Firefox, Edge, and on iPhone and iPad, including Safari. DONE

**Comment by BCcLeva** [comment]
Done.

**Comment by paulagaube** [comment]
This was an odd fix. In Prod on Dashboard > Users > Profile Fields > Agreement Terms

I edited the field, added a colon on the title "Agreement Terms:" then clicked "Update" to resave the field. 
The link to "Terms of Use" then appeared on the Sign Up page. 

Then I edited the field to remove the colon, updated, and the Terms of Use link disappeared. Edited the field and returned the colon, updated, and the link reappeared on the Sign Up page. 

So for now, the colon stays. :)
DONE on Prod
